include/mbedtls/dhm.h

```diff
@@ -1,10 +1,18 @@
 /**
  * \file dhm.h
  *
- * \brief Diffie-Hellman-Merkle key exchange
+ * \brief Diffie-Hellman-Merkle key exchange.
+ *
+ * <em>RFC-3526: More Modular Exponential (MODP) Diffie-Hellman groups for 
+ * Internet Key Exchange (IKE)</em> defines a number of standardized 
+ * Diffie-Hellman groups for IKE. 
+ *
+ * <em>RFC-5114: Additional Diffie-Hellman Groups for Use with IETF 
+ * Standards</em> defines a number of standardized Diffie-Hellman 
+ * groups that can be used. 
```

Comment:
Please remove the reference to RFC-5114.

Reply:
Agreed to delay to a future release.